src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobSignatureVerifier.java

```diff
@@ -4,33 +4,40 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.blobrouter.exceptions.DocSignatureFailureException;
+import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidConfigException;
 import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidZipArchiveException;
 import uk.gov.hmcts.reform.blobrouter.util.zipverification.ZipVerifiers;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
+import java.util.Base64;
 import java.util.zip.ZipInputStream;
 
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
 import static org.slf4j.LoggerFactory.getLogger;
 
 @Component
 public class BlobSignatureVerifier {
 
     private static final Logger logger = getLogger(BlobSignatureVerifier.class);
 
-    private final String publicKeyDerFilename;
+    private final String publicKeyBase64;
 
     public BlobSignatureVerifier(
         @Value("${public-key-der-file}") String publicKeyDerFilename
     ) {
-        this.publicKeyDerFilename = publicKeyDerFilename;
+        try {
+            publicKeyBase64 = Base64.getEncoder().encodeToString(toByteArray(getResource(publicKeyDerFilename)));
```

Comment:
is it ok time to start using `Base64.getMimeEncoder()` instead?

Reply:
Let's leave 'RFC4648 vs RFC2045' for a separate PR ;)